replit_river/client_transport.py

```diff
@@ -144,15 +156,23 @@ async def _establish_new_connection(
                 except RiverException as e:
                     await ws.close()
                     raise e
-            except Exception:
+            except Exception as e:
+                last_error = e
                 backoff_time = rate_limit.get_backoff_ms(client_id)
                 logger.exception(
-                    f"Error connecting, retrying with {backoff_time}ms backoff"
+                    (
+                        f"Error connecting: {str(e)}, "
+                        f"retrying with {backoff_time}ms backoff"
+                    )
```

Comment:
do we _really_ need this? `logger.exception` should already add `e` into the message.